test/test.ts

```diff
@@ -572,10 +591,33 @@ test('is.generator', t => {
 	testType(t, 'generator');
 });
 
+test('is.asyncGenerator', t => {
+	testType(t, 'asyncGenerator');
+
+	const fixture = (async function * () {
+		yield 4;
+	})();
+	if (is.asyncGenerator(fixture)) {
+		t.true(is.function_(fixture.next));
+	}
+});
+
 test('is.generatorFunction', t => {
 	testType(t, 'generatorFunction', ['function']);
 });
 
+test('is.asyncGeneratorFunction', t => {
+	testType(t, 'asyncGeneratorFunction', ['function']);
+
+	const fixture = async function * () {
```

Comment:
same

Reply:
I did the same test as for asyncFunction (old L564, new L583). It's to keep the same logic.